--- jadetex-3.13/Makefile.wiget	2005-02-18 12:46:46.000000000 +0100
+++ jadetex-3.13/Makefile	2005-02-18 12:49:55.000000000 +0100
@@ -11,9 +11,10 @@
 
 install: jadetex.fmt pdfjadetex.fmt
 	(TT=$$DESTDIR/`kpsewhich -expand-var '$$TEXMFMAIN'`; \
-	mkdir -p $$TT/web2c; \
-	cp pdfjadetex.fmt $$TT/web2c; \
-	cp jadetex.fmt $$TT/web2c; \
+	TV=$$DESTDIR/`kpsewhich -expand-var '$$TEXMFSYSVAR'`; \
+	mkdir -p $$TV/web2c; \
+	cp pdfjadetex.fmt $$TV/web2c; \
+	cp jadetex.fmt $$TV/web2c; \
 	mkdir -p $$TT/tex/jadetex ; \
 	cp dsssl.def jadetex.ltx $$TT/tex/jadetex)
 
